lib/libc/stdlib/getenv.3

```diff
@@ -79,6 +81,17 @@ to by the
 function.
 .Pp
 The
+.Fn secure_getenv
+function wraps the
+.Fn getenv
+function. As such, it will return
+.Fa NULL
+if the current process is tainted by uid or gid changes. Otherwise, it
```

Comment:
Please start new sentences on a new line (due to a quirk in mandoc).
Please consider using igor (pkg install igor) and mandoc -Tlint to verify the changes.
You should have two warnigs about the useless style macro Tn, unrelated to this change.
Also set the .Dd at the top to be today's date.